components/search/SearchInterface.vue

```diff
@@ -33,6 +33,13 @@
             :selected="selectedFacets[facet.name]"
             @changed="changeFacet"
           />
+          <button
+            v-if="Object.keys(selectedFacets).length"
```

Comment:
this will also show the button when only the content tier toggle is active, but the button won't do anything.

Reply:
if the count is 1 we could check `contentTierActiveState` which should be available as a computed porperty here.

Reply:
...in fact I reckon this should check for only any of the defaultFacets values being in 'this.selectedFacets.' So probably will need it's own computed property or method.